packages/phpactor.rb

```diff
@@ -0,0 +1,32 @@
+require 'package'
+
+class Phpactor < Package
+  description 'PHP completion, refactoring and introspection tool.'
+  homepage 'https://phpactor.readthedocs.io/en/develop/'
+  version '0.15.0'
+  source_url 'https://github.com/phpactor/phpactor/archive/0.15.0.tar.gz'
+  source_sha256 '02f63e77aa435874a3e8ef11a293eb24a7ee259317dfaae3f058d8bc24e39e24'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'composer'
+  depends_on 'php73'
```

Comment:
Probably don't need the php dependency unless that is a minimum version requirement since composer already has a php requirement.

Reply:
Indeed, as said in the PR description, it requires php >=7.3. I don't know how to express a minimum version so I chose a `php73` dependency

Reply:
This could be a bit tricky then.  If somebody already has another version of php installed, it might bypass the requirement altogether.  We would have to test it but I imagine if the requirement isn't met, some message should be output and the install should abort if the minimum requirement isn't met.

Reply:
I removed `phpactor` and my `php73` version and installed `php72` then ran `crew install phpactor`. It tryed to install `php73` but failed and stopped `phpactor` installation. Then I removed `php72`.

1. First problem : when installing `phpactor` it wanted to install `php74` and `php73`... Strange...
2. Second problem : installation fails if I install `php74` and then `phpactor`. `php73` is a minimal requirement, this would imply to create a new syntax in `crew` such as `depends_on php => "7.3"` as done in `chromebrew` : 
> Some Requirements can also take a string specifying their minimum version that the formula depends on.

Reply:
1. is probably happening since composer has a php74 dependency if php is not installed.  Then the phpactor has a dependency of php73 which would make sense.  We might be able to do something like below:
```
php73 = `crew search php73 2> /dev/null`.chomp
php74 = `crew search php74 2> /dev/null`.chomp
depends_on 'php73' unless "#{php73}" != "" or "#{php74}" != ""
```
If any other version of php is installed, it will output the message php x.x is already installed.

Reply:
This doesn't work : if I install `php72`, `phpactor` is installed but fails while running since `php73` minimal dependency is not fullfilled. Maybe try something based on checking if `php73` is in the installed packages list using `grep` and `crew list installed`. I am going to check that

Reply:
```
  php73 = `crew list installed | grep "php73" 2> /dev/null`.chomp
  php74 = `crew list installed | grep "php74" 2> /dev/null`.chomp
  depends_on 'php73' unless "#{php73}" != "" and "#{php74}" != ""
```
This works. But :
1- if php < 7.3 is installed  --> Fails : normal
2- if php73 is installed --> Success
3- if php74 is installed --> Success
4- if no php is installed --> Fails.

4 is a problematic case : since `composer` requires `php74` and `phpactor` by default depends on `php73`. So, crew will ask to install `php73` AND `php74` which fails. Maybe we should handle this case in the logic to depend on `php74` if no php is installed ?

Reply:
OK :  #cfc5566 solves this logic problems even if no php version is installed.

Reply:
I changed the logic to display the error message and abort installation